ramalama/cli.py

```diff
@@ -1070,3 +1071,50 @@ def inspect_cli(args):
     args.pull = "never"
     model = New(args.MODEL, args)
     model.inspect(args)
+
+
+def main():
+    parser, args = init_cli()
+
+    try:
+        import argcomplete
+        argcomplete.autocomplete(parser)
+    except Exception:
+        pass
+
+    try:
+        ModelStoreImport(args.store).import_all()
+    except Exception as ex:
+        print(f"Failed to import models to new store: {ex}")
```

Comment:
just glancing at the code, these two exceptions being swallowed (first silently, then with a log message) make be doubtful. If it's legit, a comment in the code could tell why it's safe

Reply:
It's difficult for me to speak to the details of why. This is simply the current implementation used by the library in bin/ramalama. It wasn't introduced by this PR.

Reply:
I think this is a bug, it should be exited with error code.
@engelmi PTAL